plugins/dashboard/frontend/src/app/stores/ExplorerStore.tsx

```diff
@@ -15,6 +15,7 @@ import {Link} from 'react-router-dom';
 import {RouterStore} from "mobx-react-router";
 
 export const GenesisMessageID = "1111111111111111111111111111111111111111111111111111111111111111";
+export const GenesisTransactionID = "1111111111111111111111111111111111111111111111111111111111111111"

```

Comment:
is there such a thing as genesis tx? I thought we only have genesis message and output.